stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -108,6 +108,8 @@ openblas_get_config() = strip(unsafe_string(ccall((@blasfunc(openblas_get_config
     set_num_threads(n)
 
 Set the number of threads the BLAS library should use.
+If n <= 0 then will make no change.
```

Comment:
Code quote maybe? Is this the same for MKL? And maybe a bit more descriptive like "keep the current number of threads"?

Reply:
```suggestion
If `n <= 0` then the number of threads will not be changed.
```

I am not sure where to check MKL

Reply:
https://stackoverflow.com/questions/60237626/will-mkl-set-num-threads-upper-bound-to-the-number-of-cpu-threads/60298394#60298394